terraso_backend/apps/auth/providers.py

```diff
@@ -149,3 +149,57 @@ def _build_client_secret(self):
             algorithm=self.JWT_ALGORITHM,
             headers=jwt_header,
         )
+
+
+class MicrosoftProvider:
+    OAUTH_BASE_URL = (
```

Comment:
question: should this be `BASE_URI` instead of `BASE_URL` to be consistent?

Reply:
Yes, good point. I will change this.